packages/core/useEventSource/index.ts

```diff
@@ -115,8 +115,8 @@ export function useEventSource<Events extends string[]>(
   events: Events = [] as unknown as Events,
   options: UseEventSourceOptions = {},
 ): UseEventSourceReturn<Events> {
-  const event: Ref<string | null> = deepRef(null)
-  const data: Ref<string | null> = deepRef(null)
+  const event: Ref<string | null> = shallowRef(null)
+  const data: Ref<unknown> = deepRef(null)
```

Comment:
generally **I prefere** `const data = deepRef<unknown>(null)`  i think we can even drop the `null` >  `const data = deepRef<unknown>()`

Reply:
this and your other comment are just because its a lint fix up. i wouldn't have written it this way myself

ill update

Reply:
I know. 😄 

Just wanted to mention if we touch this anyway 👍🏽